cmd/kube-controller-manager/app/core.go

```diff
@@ -238,16 +238,16 @@ func startCloudNodeLifecycleController(ctx ControllerContext) (http.Handler, boo
 
 func startRouteController(ctx ControllerContext) (http.Handler, bool, error) {
 	if !ctx.ComponentConfig.KubeCloudShared.AllocateNodeCIDRs || !ctx.ComponentConfig.KubeCloudShared.ConfigureCloudRoutes {
-		klog.Infof("Will not configure cloud provider routes for allocate-node-cidrs: %v, configure-cloud-routes: %v.", ctx.ComponentConfig.KubeCloudShared.AllocateNodeCIDRs, ctx.ComponentConfig.KubeCloudShared.ConfigureCloudRoutes)
+		klog.InfoS("Will not configure cloud provider routes for allocate-node-cidrs, configure-cloud-routes", "allocateNodeCIDR", ctx.ComponentConfig.KubeCloudShared.AllocateNodeCIDRs, "configureCloudRoutes", ctx.ComponentConfig.KubeCloudShared.ConfigureCloudRoutes)
 		return nil, false, nil
 	}
 	if ctx.Cloud == nil {
-		klog.Warning("configure-cloud-routes is set, but no cloud provider specified. Will not configure cloud provider routes.")
+		klog.InfoS("configure-cloud-routes is set, but no cloud provider specified. Will not configure cloud provider routes.")
```

Comment:
Could you remove the periods from error messages as per the [migration guide](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message)?
```suggestion
		klog.InfoS("configure-cloud-routes is set, but no cloud provider specified. Will not configure cloud provider routes")
```